book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
I suggest that rentals shouldn't just have getters but also setters since you are not just reading but also writing to it👍🏼.
```suggestion
  attr_accessor :title, :author, :rentals
```